doc/gallery/index.md

```diff
@@ -251,27 +251,26 @@ The gluoNNet aviation tracking application visualizes airplane positioning (ADS-
 
 ::::
 
-
 ```{toctree}
 :titlesonly:
 :hidden:
 :maxdepth: 1
 
-portfolio_optimizer
-streaming_videostream
-windturbines
-portfolio_analyzer
-glaciers
-vtk_slicer
-nyc_deckgl
-gapminders
-vtk_interactive
-penguin_crossfilter
+altair_brushing
```

Comment:
It will help navigating a lot of the menu is sorted.

![image](https://github.com/holoviz/panel/assets/42288570/b609e471-631f-4d25-a424-f28a92492f06)